eatngo-infra/src/main/kotlin/com/eatngo/aws/s3/S3Config.kt

```
@@ -0,0 +1,27 @@
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.s3.S3Client
+import software.amazon.awssdk.services.s3.presigner.S3Presigner
+
+@Configuration
+class S3Config(
+//    @Value("\${cloud.aws.region.static}")
+//    private val region: String
+) {
```

detekt (1.23.8)

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)